resources/windows/wix/service.mcollective.wxs.erb

```diff
@@ -0,0 +1,40 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Fragment>
+    <ComponentGroup Id="Service_MARIONETTECOLLECTIVE">
```

Comment:
Okay, I'm seeing some discrepancies. In https://github.com/puppetlabs/vanagon/commit/77664321791beeb01d29ebdf3cbb941a78b669e0, we set `:id => "#{service_name.gsub(/([^A-Za-z0-9])/, '').upcase}"` but then when we reference that in the wix files, we prepend `Service_` to it. Which is it? `MARIONETTECOLLECTIVE` or `Service_MARIONETTECOLLECTIVE`? Are we ever going to need `service.id` to be `MARIONETTECOLLECTIVE` instead of `Service_MARIONETTECOLLECTIVE`?